frontend/components/Domain/Household/GroupMealPlanRuleForm.vue

```diff
@@ -148,6 +148,27 @@ export default defineComponent({
         label: i18n.tc("general.date-updated"),
         type: "date",
       },
+      {
```

Comment:
These have to be snake_case otherwise they end up being invalid on the frontend

`nutrition.carbohydrate_content`, etc.

When I try adding this to my meal plan rules, and I go to edit the rule again, it gets removed